package.json

```diff
@@ -21,6 +21,7 @@
     "ejs": "^3.1.6",
     "express": "^4.16.4",
     "file-loader": "^3.0.1",
+    "gm": "^1.23.1",
```

Comment:
Make this an optional dependency please. I really do not like gm/imagemagick much. And it's a huge dependency.
https://docs.npmjs.com/cli/v7/configuring-npm/package-json#optionaldependencies